libnopegl/src/path.c

```diff
@@ -131,6 +131,11 @@ int ngli_path_bezier3_to(struct path *s, const float *ctl0, const float *ctl1, c
     return add_segment_and_move(s, &segment, to);
 }
 
+int ngli_path_close(struct path *s)
+{
+    return ngli_path_line_to(s, s->origin);
```

Comment:
`s->origin` is introduced in a later commit.

Reply:
Ah my bad, move that commit later. Also tested each individual commit to make sure it didn't happen elsewhere.